packages/yew/src/virtual_dom/mod.rs

```diff
@@ -79,20 +80,23 @@ mod feat_ssr_hydration {
         pub fn open_start_mark(&self) -> &'static str {
             match self {
                 Self::Component(_) => "<[",
+                Self::Raw => "<#",
                 Self::Suspense => "<?",
             }
         }
 
         pub fn close_start_mark(&self) -> &'static str {
             match self {
                 Self::Component(_) => "</[",
+                Self::Raw => "</#",
                 Self::Suspense => "</?",
             }
         }
 
         pub fn end_mark(&self) -> &'static str {
             match self {
                 Self::Component(_) => "]>",
+                Self::Raw => "#>",
```

Comment:
```suggestion
                Self::Raw => ">",
```

After seeing the test case, I think the end mark should be emitted as `>` and not `#>` so raw can be emitted as `<!--<#>--><!--</#>-->` instead of `<!--<##>--><!--</##>-->`.

Reply:
Ah, you're right! Seems like `Suspense` also uses `>` as the end mark so that makes sense. Fixed!